### DISCUSSION TITLE
Linux, a labor of love?

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#360] [General]
Unless your distro is in the top 5, what else can it be ?

**Comment by geckolinux** [comment]
In my case, I've always admired the Debian project for a lot of reasons, but found it atrocious to use and configure for normal desktop usage out-of-the-box. So for my own needs and for some users that I support I created SpiralLinux as a way to take care of all of the initial configuration and setup tasks, some of which a "normal" user often can't figure out when dealing with vanilla Debian.

**Reply by dacomboman** [reply]
Having read your appreciation for Cinnamon desktop, decided to give it a try.
Personally, do find it much snappier than Mint's iteration and even MX's XFCE!
Being a distro-hopper, and mostly a KDE fan, never thought i'd be happy with anything else.
Your SpiralLinux Cinnamon ed. is growing on me from day to day! :+1: 

Thank you!

**Reply by AlphaElwedritsch** [reply]
I've already made the journey via MX (xfce) and Mint (LMDE). Ultimately took the same path. And yes, the feeling of having found the right distro and having done everything right grows day by day for me, too 😉
But actually I'm more a budgie then cinnamon spicy ;)

**Reply by geckolinux** [reply]
@AlphaElwedritsch and @dacomboman Great to hear, thanks for saying so!

**Comment by AlphaElwedritsch** [comment]
And I thank you for that.
That you don't just do this for yourself and for the users you support. But you make it available to all of us.
As already mentioned. SpiralLinux is exactly what I've been looking for for a long time but never found.
And I am happy and grateful that I have found it now.
